pymatgen/entries/compatibility.py

```diff
@@ -888,7 +910,7 @@ def __init__(
             self.u_corrections = {}
             self.u_errors = {}
 
-    def get_adjustments(self, entry: Union[ComputedEntry, ComputedStructureEntry]):
```

Comment:
Noting the type hint is removed here, was there a reason?

Reply:
Nope, I think that was probably an accident 🙃. Restored.